solutions/gold/usaco-925.mdx

```diff
@@ -0,0 +1,127 @@
+---
+id: usaco-925
+source: USACO Platinum 2019 February
+title: Mooriokart
+author: Justin Ji
+---
+
+[Official Editorial (C++)](https://usaco.org/current/data/sol_mooriokart_platinum_feb19.html)
+
+<Spoiler title = "Hint 1">
+
+The graph is a forest, and we are only allowed to pick one path from each tree
+to reach our total. How can we write a knapsack DP to find our answer?
```

Comment:
i mean is this not just restating the problem

Reply:
true, i mainly added it to lean into hint 2, but I might end up removing it later

Reply:
agree - just combine the two hints into one

Reply:
ok i combined them - lmk if it needs a bit more work